dig/chain.json

```diff
@@ -53,6 +53,10 @@
       }
     ]
   },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dig/images/dig.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dig/images/dig.svg"
```

Comment:
IMO that dig svg is way too different than the PNG and should probably be omitted